definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js

```diff
@@ -3130,13 +3128,10 @@ declare module "lodash/fp" {
     defaultTo<T1: number, T2>(defaultValue: T2, value: T1): T1 | T2;
     defaultTo<T1: void | null, T2>(defaultValue: T2): (value: T1) => T2;
     defaultTo<T1: void | null, T2>(defaultValue: T2, value: T1): T2;
-    flow: $ComposeReverse;
-    flow(funcs: Array<Function>): Function;
-    pipe: $ComposeReverse;
-    pipe(funcs: Array<Function>): Function;
-    flowRight: $Compose;
-    flowRight(funcs: Array<Function>): Function;
-    compose: $Compose;
+    flow: ($ComposeReverse & (funcs: Array<Function>) => Function);
+    pipe: ($ComposeReverse & (funcs: Array<Function>) => Function);
```

Comment:
Pipe actually doesn't exist either.

Reply:
Same as my previous comment, this is for `lodash/fp` and `pipe` is basically an alias `flow`.

Reply:
Neat! I wasn't aware that there are some aliased functions. IN this case everything seems good!

Reply:
S no longer `needs-udpates`? ;)

Reply:
@omeid yep! Unfortunately I'm still lacking rights for the repo so there is not much I can do except pinging @AndrewSouthpaw :D